atc/plan.go

```diff
@@ -5,6 +5,7 @@ type Plan struct {
 	Attempts []int  `json:"attempts,omitempty"`
 
 	Aggregate *AggregatePlan `json:"aggregate,omitempty"`
+	Parallel  *ParallelPlan  `json:"parallel,omitempty"`
```

Comment:
:thinking: The name feels slightly off; most of them are adjectives or verbs (except `task` I guess).

What do you think of `in_parallel`? I kinda like that as it fits alongside `on_success`/`do`/etc.

Reply:
Makes sense, I've changed it to `in_parallel` 👍